### DISCUSSION TITLE
Styling for different devices

### DISCUSSION THREAD

**Discussion by srivatsan2607** [GrapesJS/grapesjs#4706] [Q&A]
Hi @artf, 
I am trying to use a uniform styling for all type of devices but the styles are different based on the type of device as the styles implemented in phone device is not applicable to laptop device. 
It seems like users have to style for different devices even if the same styling is needed.
Is there any solution where by default the styles are the same on all devices and if needed can be changed?
Also is there any way to implement responsiveness as the elements in phone size will auto-adjust themselves to the laptop device?

**Comment by artf** [comment]
I'm not sure what you're talking about, by default, if you apply a style on desktop you will see that available also on other devices. Maybe you can link a reproducible demo of your issue.
